packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -4,6 +4,14 @@ spec:
   addonComponents:
     istiocoredns:
       enabled: {{ .Values.istiocoredns.enabled }}
+    prometheus:
+      enabled: false
+    grafana:
+      enabled: false
+    kiali:
+      enabled: false
+    tracing:
+      enabled: false
```

Comment:
Why'd these come back ?

Reply:
This pr shows them getting removed only when going to 1.7 https://github.com/rancher/charts/pull/581/files. We had them in as they still had default values in 1.6 so we wanted to ensure they were always false.

Reply:
Guess it doesn't matter if we are deleting 1.6 anyway